assets/dev/js/frontend/frontend.js

```diff
@@ -320,6 +319,16 @@ export default class Frontend extends elementorModules.ViewModule {
 		} );
 	}
 
+	populateActiveBreakpointsConfig() {
```

Comment:
From which branch is it?

Reply:
I actually don't have any idea. Probably came from some merge.